ui/design/src/components/Editor/index.tsx

```diff
@@ -172,11 +172,12 @@ const EditorBox: React.FC<IEditorBox> = React.forwardRef((props, ref) => {
   /**
    * initialise editor with all the required plugins
    */
-  const editor = useMemo(
-    () => withLinks(withTags(withMentions(withImages(withHistory(withReact(createEditor())))))),
-    [],
+  const editorRef = useRef(
+    withLinks(withTags(withMentions(withImages(withHistory(withReact(createEditor())))))),
```

Comment:
why switch to useRef here?

Reply:
To signal that the editor instance does not change between renders

Reply:
wasn't useMemo providing the same utility?

Reply:
only when using useMemo with empty deps, but apparently codefactor does not like it